src/legacy/core_plugins/console/public/np_ready/application/components/editor_content_spinner.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EuiLoadingContent, EuiPageContent } from '@elastic/eui';
+import React from 'react';
+
+export const EuiLoadingContent10 = () => {
```

Comment:
This feels like an odd name (though I see why it's like that) - will there be `EuiLoadingContent5`, `EuiLoadingContent20`?

Reply:
Will clean up this name :)